packages/hub/src/users.spec.ts

```diff
@@ -101,16 +105,100 @@ describe('All apis...', () => {
         expect(root).to.not.be.undefined
 
         // Ensure context is set properly
-        expect(users.context.toJSON()).to.have.ownProperty('x-textile-thread-name')
-        expect(users.context.get('x-textile-thread-name')).to.equal('my-buckets')
+        expect(db.context.toJSON()).to.have.ownProperty('x-textile-thread-name')
+        expect(db.context.get('x-textile-thread-name')).to.equal('my-buckets')
 
         // We should have a thread named "my-buckets"
-        const res = await users.getThread('my-buckets')
-        expect(res.id).to.deep.equal(users.context.toJSON()['x-textile-thread'])
+        const res = await db.getThread('my-buckets')
+        expect(res.id).to.deep.equal(db.context.toJSON()['x-textile-thread'])
 
         // The dev should see that the key was used to create one thread
         // @todo: Use the cli to list keys
       })
+
+      it('should check the users own usage', async function () {
+        if (isBrowser) return this.skip()
```

Comment:
why not in browser? too slow?

Reply:
it's relying on counts of things done in node tests